lib/pbench/test/unit/server/test_datasets_inventory.py

```diff
@@ -127,6 +128,6 @@ def mock_send_file(path_or_file, *args, **kwargs):
 
         file_content, args = mock_args
 
-        assert str(file_content) == "file_as_a_byte_stream"
+        assert str(file_content).startswith("<_io.BytesIO")
```

Comment:
This is a fascinating assertion.  😆

Please don't do this.  🙂  If you want to check whether `file_content` is an instance of `io.BytesIO`, please use `isinstance()`.  😁 

However, what is this test case trying to exercise?  (What's the "unit"?)  Since we have a mock for the call to `send_file()` in `datasets_inventory`, I'm guessing that the unit is `DatasetsInventory._get()`.  If that's the case, then we should be mocking its call to `CacheManager.filestream()` which would allow you to control what the stream is, which in turn would allow you to assert that it has the correct value here (without resorting to doing partial comparisions of stringized type names 🙃).